app/actions/index.js

```diff
@@ -134,8 +128,6 @@ export const downloadSparseDat = ({ key }) => dispatch => {
     dispatch(updateState(dat))
     joinNetwork(dat)(dispatch)
     updateConnections(dat)(dispatch)
-
-    dats[key] = { dat, path, opts }
```

Comment:
@juliangruber how do you remove this dat and close it when you click cancel download if you don't keep reference to it somewhere?

Reply:
you're right, I misread the code

Reply:
will fix!